signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java

```diff
@@ -69,6 +72,12 @@ public void onConnected(SessionConnectedEvent connectedEvent) {
     public void onDisconnected(SessionDisconnectEvent disconnectEvent) {
         String username = Objects.requireNonNull(disconnectEvent.getUser()).getName();
         log.info("WebSocket Disconnected >>>>>> {}", username);
+        if(disconnectEvent.getCloseStatus()!=null && !disconnectEvent.getCloseStatus().equals(CloseStatus.NORMAL)){
```

Comment:
if you reverse the condition you could eliminate null check @Md-Humair-KK . 
if(!CloseStatus.NORMAL.equals(disconnectEvent.getCloseStatus())) try this once.

Reply:
Yes @ase-101 Thank you for the information, it worked after inverting the condition without null check.